### PR TITLE
cb90 Adjust the responsiveness of the board

### PR DESCRIPTION
**Pull request by Brandonz07**

### CODE REVIEW

File: src/pages/board/outlets/boardlayout/styles.ts
Comment (novadeveloper14) [inline src/pages/board/outlets/boardlayout/styles.ts:1]: In the mobile version the table should occupy the entire width

image

like this

image

Comment (novadeveloper14) [inline src/pages/board/outlets/boardlayout/styles.ts:1]: When using this functionality the table should occupy all 1440px

image

Comment (novadeveloper14) [inline src/pages/board/outlets/boardlayout/styles.ts:1]: If this functionality is used the size of the table is changing and this should not happen

image

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/pages/board/outlets/boardlayout/styles.ts
Comment (Brandonz07) [reply src/pages/board/outlets/boardlayout/styles.ts:1]: Using the styled, the errors are solved

**Review by novadeveloper14 — APPROVED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**